server/fleet/mdm.go

```diff
@@ -628,3 +630,28 @@ func FilterMacOSOnlyProfilesFromIOSIPadOS(profiles []*MDMAppleProfilePayload) []
 
 // RefetchCommandUUIDPrefix is the prefix used for MDM commands used to refetch information from iOS/iPadOS devices.
 const RefetchCommandUUIDPrefix = "REFETCH-"
+
+// VPPTokenInfo is the representation of the VPP token that we send out via API.
```

Comment:
These 3 types are used because we have 3 different representations of the token:

1. The view we want to send back in the response to `GET /vpp`
2. The actual VPP token itself (i.e. the JSON itself, not the base64 encoded string)
3. The data we want to store in the DB, which includes 2 _and_ the `location` field, which comes from an Apple API

I'm open to ways to minimize the number of representations we need! 😅